lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -36,7 +36,7 @@ using namespace glow;
 /// The resulting file is included here to compile the bitcode image into our
 /// library.
 static const unsigned char libjit_bc[] = {
-#include "glow/CPU/libjit_bc.inc"
+#include "glow/CPU/libjit.inc"
```

Comment:
this most likely will break fbcode version (need corresponding changes)

Reply:
Yeah, I think that dealing with copying files is kinda silly and just complicates things, we should just fix the include paths instead.